docs/advanced/transaction-details.md

```diff
@@ -75,3 +76,210 @@ The witness data of a transaction involves only its inputs. The included data fi
+</tr>
+<tr>
+<td>Number inputs</td>
+<td><code>000002</code></td>
+<td>2</td>
+</tr>
+<tr>
+<td colspan="3" style="text-align: center;">1st input</td>
+</tr>
+<td>Transaction hash</td>
+<td style="word-break: break-all;"><code>4ff28b534361e383b5b149df4572feb1fbc659cb07067c3cb6c684452ae79f54</code></td>
+<td></td>
+</tr>
+<tr>
+<td>Output index</td>
+<td><code>01000000</code></td>
+<td>1</td>
+</tr>
+<tr>
+<td>Tree</td>
+<td><code>00</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Sequence number</td>
+<td><code>ffffffff</code></td>
+<td>4294967295</td>
+</tr>
+<tr>
+<td colspan="3" style="text-align: center;">2nd input</td>
+</tr>
+<tr>
+  <td>Transaction hash</td>
+<td style="word-break: break-all;"><code>c619e6ade3f469b8bb60b8ae9b0599eeeca05cd2db251cadea443344eafdac57</code></td>
+<td></td>
+</tr>
+<tr>
+<td>Output index</td>
+<td><code>01000000</code></td>
+<td>1</td>
+</tr>
+<tr>
+<td>Tree</td>
+<td><code>00</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Sequence number</td>
+<td><code>ffffffff</code></td>
+<td>4294967295</td>
+</tr>
+<tr>
+<td colspan="3" style="text-align: center;">1st output</td>
+</tr>
+<tr>
+<td>Output amount</td>
+<td><code>91c2e26d02000000</code></td>
+<td>104.33</td>
+</tr>
+<tr>
+<td>Version</td>
+<td><code>0000</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Public key script length</td>
+<td><code>19</code></td>
+<td>25</td>
+</tr>
+<tr>
+<td>Public key script</td>
+<td style="word-break: break-all;"><code>76a914f2ccddc70f8b8bcea24a362bf404841d53cbebc088ac</code></td>
+<td></td>
+</tr>
+<tr>
+<td colspan="3" style="text-align: center;">2nd output</td>
+</tr>
+<tr>
+<td>Output amount</td>
+<td><code>d676dd6801000000</code></td>
+<td>60.54</td>
+</tr>
+<tr>
+<td>Version</td>
+<td><code>0000</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Public key script length</td>
+<td><code>19</code></td>
+<td>25</td>
+<td></td>
+</tr>
+<tr>
+<td>Public key script</td>
+<td style="word-break: break-all;"><code>76a9141a1f5c7e9f7696989dced5cb9b61464f95790b7288ac</code></td>
+<td></td>
+</tr>
+<tr>
+<td>Lock time</td>
+<td><code>00000000</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Expiry</td>
+<td><code>00000000</code></td>
+<td>0</td>
+</tr>
+<tr>
+<td>Number witnesses</td>
+<td><code>02</code></td>
+<td>2</td>
+</tr>
+<tr>
+<td colspan="3" style="text-align: center;">1st witness</td>
+</tr>
+<tr>
+  <td>Input amount</td>
+<td><code>ca68865402000000</code></td>
+<td>100.08</td>
+</tr>
+<tr>
+<td>Block height</td>
+<td><code>a29c0400</code></td>
+<td>302242</td>
+</tr>
+<tr>
+<td>Block index</td>
+<td><code>04000000</code></td>
+<td>4</td>
+</tr>
+<tr>
+<td>Signature script length</td>
+<td><code>6a</code></td>
+<td>106</td>
+</tr>
+<tr>
+<td>Signature script</td>
+<td style="word-break: break-all;"><code>4730440220139466bd10b1f071f9b4ac16ed434d63d090613fd06470f7bca9bba2b6ea6a0a02203cad1297ab9384466b3f304cd61e7e4ffa9abc1ae9fe1630207fae6f200da1680121024d540859b805c5780f2f0da350df8757c1defb72e3381b252d20874478a5549c</code></td>
+<td></td>
```

Comment:
2 data pushes.  1st push is DER signature.  2nd push is public key.

Reply:
Just used this text as is. This is definitely helpful, though wondering, would it be useful to break this down further? E.g. separate out DER signature and public key hex values and label them? Or can we assume the reader knows how to interpret this?

Reply:
If it will fit well, that makes sense.  I was trying to keep it relatively short because the format I see on the current markdown preview is wonky.

If you want to break it down more though, it's:

2 data pushes.

| Script Operation | Value | Description|
| --- | --- | --- |
| 1st push |`3045022100da5b0fde58c4c57a88d96f83b02eddcb67b1b68ce95cc562c895f2e4e` `57bb44302201a8744aed654d2979bdd797187f6bf63ad1898c5d5c585309eeca558` `742858b201` | DER signature |
| 2nd push | `0307e3d98b004b15561c0f5aa158c90f93de33756d3bfd1f8a1058a4d90ffc7ec7` | Public key |

Reply:
Formatting is getting a little wonky, but my guess is that if you're slogging through this level of detail, you'll probably appreciate more breakdown. 

Below is a screenshot of how I've implemented it (FF),
![image](https://user-images.githubusercontent.com/1634777/57825146-7e35f600-7752-11e9-9402-4441d172924b.png)

Let me know if this text is inaccurate, or could be worded more clearly. I'm still slightly confused as to what the DER signature is. I assume it's a binary string of digits encoded with DER, which is what `asm` is in the decoded JSON (i.e. this is still encoded),

```
         "scriptSig": {
            "asm": "3045022100da5b0fde58c4c57a88d96f83b02eddcb67b1b68ce95cc562c895f2e4e57bb44302201a8744aed654d2979bdd797187f6bf63ad1898c5d5c585309eeca558742858b201 0307e3d98b004b15561c0f5aa158c90f93de33756d3bfd1f8a1058a4d90ffc7ec7",
            "hex": "483045022100da5b0fde58c4c57a88d96f83b02eddcb67b1b68ce95cc562c895f2e4e57bb44302201a8744aed654d2979bdd797187f6bf63ad1898c5d5c585309eeca558742858b201210307e3d98b004b15561c0f5aa158c90f93de33756d3bfd1f8a1058a4d90ffc7ec7"
         }
```

Reply:
That text is fine except it's just a DER signature.  No need to say asm.  asm just means it's the disassembly of the script.

There really is no need to break the DER signature down even more in this document imo.  It's a well known and defined ASN.1 encoding.

Reply:
Yes, I think we have drilled far enough. Removed `(asm)`.